pages/support.tsx

```diff
@@ -0,0 +1,301 @@
+  width: 100%;
+  margin: auto;
+  box-sizing: border-box;
+  padding: 1rem;
+  position: relative;
+  z-index: 10;
+`;
+
+const Button = styled.button`
+  background: linear-gradient(to bottom, #FDD85C 0%, #F5C927 100%);
+  color: white;
+  box-shadow: 0px 4px 4px rgba(0, 0, 0, 0.25);
+  border-radius: 0.25rem;
+  padding: 0.5rem 1rem;
+`;
+
+const SupportRow = styled.div`
+  margin: 4rem 0;
+  gap: 1rem;
+  @media (min-width: 1200px) {
+    margin: 6rem 0;
+    gap: 3rem;
+    display: flex;
+    align-items: center;
+  }
+`;
+
+const SupportImage = styled.div`
+  position: relative;
+  width: 100%;
+  @media (min-width: 1200px) {
+    max-width: 450px;
+  }
+`;
+const SupportText = styled.div`
+  position: relative;
+  z-index: 10;
+  @media (max-width: 1200px) {
+    margin-top: 1rem;
+  }
+`;
+
+const SupportBlob = styled.div`
+  width: 100%;
+  height: 130%;
+  position: absolute;
+  top: -15%;
+  left: -15%;
+  background-repeat: no-repeat;
+  background-size: contain;
+  pointer-events: none;
+`;
+
+export default function Support() {
+  return (
+    <>
+      <Head>
+        <title>Support | Holaplex</title>
+        <meta
+          property="description"
+          key="description"
+          content="Our mission is to empower creators and collectors with a suite of tools to create, market, and sell NFTs."
+        />
+        <body />
+      </Head>
+
+      <HeroWrapper>
+        <Section>
+          <Content className="text-center lg:my-8">
+            <div style={{ maxWidth: '1044px' }} className='mx-auto'>
+              <Heading>Solanas top community <br className='hidden lg:block' /> supported NFT platform</Heading>
+              <Paragraph>
+                Web3 is an exciting place but sometimes we need help. We rely on collaboration from members from the community to provide assistance. Get involved by contributing to our wiki page or in our discord.
+              </Paragraph>
+              <a href='https://docs.holaplex.com/'><Button>Read The Docs</Button></a>
+            </div>
+          </Content>
+        </Section>
+      </HeroWrapper>
+
+      <div className="container mx-auto px-6 md:px-12">
+        <Section>
+          <div className="w-full lg:w-1/2">
+            <Content>
+              <SubHeading>Need Help?</SubHeading>
+              <Heading>Get support from the community</Heading>
+            </Content>
+          </div>
+          <Content>
+            <div className='lg:flex lg:gap-4 w-full'>
+              <div>
+                <Paragraph>
+                  <b>Information at your fingertips</b><br />
+                  Our documentation is expanding by the day thanks to contributions from our community.<br />
+                  <a href="https://wiki.holaplex.com/" style={{ color: '#F7CC34' }}>Find your answer</a>
+                </Paragraph>
+              </div>
+              <div>
+                <Paragraph>
+                  <b>Get help from the community</b><br />
```

Comment:
We generally prefer to use tailwind for font-weight, e.g. 

```suggestion
                  <span className="font-bold">Get help from the community</span><br />
```